Assets/GothicVR/Scripts/Npc/Actions/AnimationActions/UseItemToState.cs

```diff
@@ -1,55 +1,88 @@
 using GVR.Caches;
 using GVR.Creator;
+using GVR.GothicVR.Scripts.Manager;
 using UnityEngine;
+using ZenKit.Daedalus;
 
 namespace GVR.Npc.Actions.AnimationActions
 {
     public class UseItemToState : AbstractAnimationAction
     {
         
-        private const string animationStartScheme = "T_{0}_STAND_2_S{1}" ;
-        private const string animationEndScheme = "T_{0}_S{1}_2_STAND" ;
-        
+        private const string animationScheme = "T_{0}_{1}_2_{2}";
```

Comment:
What is that scheme? is that hardcoded or does it come from the original?

Reply:
It's from Gothic. I rent it from OG and also double checked with Gothic animation files. The names are visible inside loaded .anim assets from ZK.

Reply:
Documentation is something for the future. ;-)